src/nav/synthetics.yml

```diff
@@ -117,10 +119,10 @@ pages:
         path: /docs/synthetics/synthetic-monitoring/pages/synthetic-monitoring-troubleshoot-downtime
       - title: SLA report
         path: /docs/synthetics/synthetic-monitoring/pages/synthetic-monitoring-aggregate-monitor-metrics
-  - title: Synthetics API
+  - title: API
```

Comment:
I would clarify which API this is, and maybe even break it out of its own category. Feels overkill to have a drop down for one doc

Reply:
it was like that before, and mainly serves to point people to docs elsewhere. i wasn't going to try to improve that with this jira